source/common/common/absl_mutex.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "common/common/thread_annotations.h"
+
+#include "absl/synchronization/mutex.h"
+
+namespace Envoy {
+namespace Locking {
```

Comment:
Maybe this could be `Thread` namespace as with `MutexBasicLockable` in `thread.h`? (Or move `MutexBasicLockable` under `Locking` or another new namespace?

Reply:
done; removed the new file absl_mutex.h and just put the content in thread.h.  Much better.